src/clj/puppetlabs/services/jruby/jruby_internal.clj

```diff
@@ -1,16 +1,16 @@
-(ns puppetlabs.services.jruby.jruby-puppet-internal
+(ns puppetlabs.services.jruby.jruby-internal
   (:require [schema.core :as schema]
-            [puppetlabs.services.jruby.jruby-puppet-schemas :as jruby-schemas]
+            [puppetlabs.services.jruby.jruby-schemas :as jruby-schemas]
             [clojure.tools.logging :as log]
             [puppetlabs.kitchensink.core :as ks])
-  (:import (com.puppetlabs.puppetserver.pool JRubyPool)
-           (puppetlabs.services.jruby.jruby_puppet_schemas JRubyPuppetInstance PoisonPill ShutdownPoisonPill)
+  (:import (com.puppetlabs.jrubyutils.pool JRubyPool)
```

Comment:
I think it might be slightly more consistent to name this `com.puppetlabs.jruby_utils`, but I don't care very much :)